### DISCUSSION TITLE
Send as a different email address

### DISCUSSION THREAD

**Discussion by BlueFyre** [berrystephenw/OctoText#118] [Q&A]
Hi, is there any way to configure the email that is being sent from vs the email that is used to authenticate?

**Comment by berrystephenw** [comment]
Apologies for not responding sooner. There is no way currently to do this without using the new (and not yet released) API and writing your own plugin. Partly because I've seen this fail in testing when the from address doesn't match the address used to log into the mail server, and partly because it could be used to spam people. I also really want to cut down on the confusion in setting up the plugin and this would add more fields in the settings dialog. 

Is there a reason why you would want to do this?
